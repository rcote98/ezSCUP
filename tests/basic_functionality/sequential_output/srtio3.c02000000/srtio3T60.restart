2	2	2
5	3
Sr	Ti	O
3.00000000E-02	3.00000000E-02	0.00000000E+00	0.00000000E+00	0.00000000E+00	0.00000000E+00
0	0	0	1	1	-5.39520330E-02	6.06530109E-02	2.79000509E-02
0	0	0	2	2	-1.00932724E-01	1.16719172E-01	1.84179501E-03
0	0	0	3	3	2.98081822E-01	5.79395274E-02	-1.17208953E-02
0	0	0	4	3	-4.95205315E-02	-5.28225406E-02	-1.98545246E-01
0	0	0	5	3	5.34030741E-02	2.50866773E-02	-1.90704333E-01
0	0	1	1	1	-4.88934080E-02	6.77394934E-02	4.28592793E-03
0	0	1	2	2	-1.18241456E-01	1.13160785E-01	2.06149445E-03
0	0	1	3	3	-1.30776506E-01	-2.86742678E-01	-5.48889547E-03
0	0	1	4	3	5.69475818E-02	-5.70143802E-02	1.61140990E-01
0	0	1	5	3	4.44390215E-02	-4.66080344E-02	1.59416190E-01
0	1	0	1	1	-3.79562818E-02	7.80084042E-02	1.64971540E-02
0	1	0	2	2	-9.41146037E-02	1.25383711E-01	1.07325781E-02
0	1	0	3	3	-1.03818692E-01	-3.01111499E-01	-8.73551498E-03
0	1	0	4	3	5.87316850E-02	-7.05285592E-02	1.66399049E-01
0	1	0	5	3	6.71875258E-02	-6.30511963E-02	1.90730074E-01
0	1	1	1	1	-7.17034828E-02	6.16677249E-02	1.75435907E-02
0	1	1	2	2	-9.25708457E-02	1.24661450E-01	1.50132341E-02
0	1	1	3	3	3.08646356E-01	8.17019996E-02	1.21467441E-02
0	1	1	4	3	-2.98918003E-02	-4.66519273E-02	-1.49185199E-01
0	1	1	5	3	3.04625612E-02	2.64451647E-02	-1.87877746E-01
1	0	0	1	1	-5.15372104E-02	8.52689140E-02	-5.36659169E-02
1	0	0	2	2	-1.10169548E-01	1.09124277E-01	7.07870562E-03
1	0	0	3	3	-1.00608808E-01	-3.15962882E-01	-1.15615798E-02
1	0	0	4	3	6.02835262E-02	-5.94299279E-02	2.26312306E-01
1	0	0	5	3	5.44639297E-02	-7.31254134E-02	2.01763510E-01
1	0	1	1	1	-5.44707602E-02	2.48046190E-02	8.30957746E-03
1	0	1	2	2	-1.19913678E-01	1.17884009E-01	-5.38072945E-03
1	0	1	3	3	2.83018640E-01	1.85211072E-01	-1.07612699E-02
1	0	1	4	3	-9.06516033E-03	-4.62418208E-02	-2.35887906E-01
1	0	1	5	3	5.26584456E-02	-2.46380041E-02	-1.99121209E-01
1	1	0	1	1	-5.68287385E-02	2.82598785E-02	8.78895122E-03
1	1	0	2	2	-1.07654139E-01	1.03652229E-01	-1.63279685E-02
1	1	0	3	3	2.68215928E-01	1.48583229E-01	-4.31865024E-03
1	1	0	4	3	-1.75779222E-03	-4.93922292E-02	-2.31134603E-01
1	1	0	5	3	4.73246255E-02	1.15522066E-02	-1.67912709E-01
1	1	1	1	1	-3.40230065E-02	5.82049526E-02	-1.17329239E-02
1	1	1	2	2	-8.08831356E-02	1.25016400E-01	-1.32211646E-02
1	1	1	3	3	-1.12915869E-01	-3.47717530E-01	8.13491338E-03
1	1	1	4	3	5.27964893E-02	-5.95364926E-02	2.68136374E-01
1	1	1	5	3	3.55389983E-02	-3.61537950E-02	1.99051248E-01
