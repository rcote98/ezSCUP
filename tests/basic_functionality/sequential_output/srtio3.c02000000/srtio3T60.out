
*****************************************************************
*                          Welcome to                           *
*                           SCALE-UP                            *
*                                                               *
*            Second-Principles Computational Approach           *
*                             for                               *
*                     Lattice and Electrons                     *
*                                                               *
*                       Version:         1.0.0.Gorbea           *
*                    Build date: 20200801T181                   *
*****************************************************************
*                    SCALE-UP is property of                    *
*              the University of Cantabria (Spain)              *
*    and the Luxembourg Institute of Science and Technology     *
*****************************************************************
* Authors:                                                      *
*                                                               *
*  - Pablo Garcia-Fernandez   garciapa@unican.es                *
*  - Jorge Iñiguez            jorge.iniguez@list.lu             *
*  - Javier Junquera          junqueraj@unican.es               *
*****************************************************************
* SCALE-UP is based on two earlier programs:                    *
*                                                               *
* -  SPLD ('Second-Principles Lattice Dynamics'), developed     *
*    at the ICMAB-CSIC (Spain) by J.C. Wojdel, M.P. Ljungberg   *
*    and J. Iñiguez, and distributed under MIT license.         *
*                                                               *
* -  MKII a second-principles electron code                     *
*    developed at the University of Cantabria by                *
*    P. Garcia-Fernandez  and J. Junquera.                      *
*****************************************************************
*                                                               *
* Bibliography:                                                 *
*                                                               *
*  P. Garcia-Fernandez et al. Phys. Rev. B 93, 195137 (2016)    *
*  J. C. Wojdel et al., J. Phys. Cond. Matt. 25, 305401 (2013)  *
*****************************************************************
Reading parameter file... lattice... done!
Reading restart file     srtio3T60.restart                                                               

Basic data:
  Number of CPUs:               1
  Run mode:            Montecarlo
  Lattice:                 Active
  Electrons:           NOT Active
  External electric field, static. Field components (V/m):   0.000000E+00   0.000000E+00   0.000000E+00

Reference system:
 Supercell data:
  Lattice constants (Ang):   7.690   7.690   7.690
  Lattice angles (degree):  90.000  90.000  90.000
 Supercell size:   2   2   2

 Total number of atoms          :      40
 Total number of orbitals       :       0
 Total number of active orbitals:       0

 Atom in unit cell:
 Atom   Label       Mass  Charge 
    1   Sr         87.60     0.0
    2   Ti         47.90     0.0
    3   O          16.00     0.0
    4   O          16.00     0.0
    5   O          16.00     0.0

 Orbitals in unit cell:
 Orbital   Label  

 Background dielectric tensor:
    6.2319     0.0000     0.0000
    0.0000     6.2319    -0.0000
    0.0000    -0.0000     6.2319

Initial values (basic output MC simulation)
Energy (eV):            -0.298474E+05
Polarization (C/m2):    -0.193186    0.219497    0.003823
Strain:                  0.030000    0.030000    0.000000    0.000000    0.000000    0.000000

LT:    Iter       Temp(K)    Energy(eV)   Pol_x(C/m2)   Pol_y(C/m2)   Pol_z(C/m2) Av_Pl_x(C/m2) Av_Pl_y(C/m2) Av_Pl_z(C/m2)       Strn_xx       Strn_yy       Strn_zz       Strn_yz       Strn_xz       Strn_xy    Av_Strn_xx    Av_Strn_yy    Av_Strn_zz    Av_Strn_yz    Av_Strn_xz    Av_Strn_xy 
LT:      10  0.600000E+02 -0.298473E+05 -0.219147E+00  0.186268E+00  0.145488E-01 -0.233179E+00  0.221068E+00  0.151640E-01  0.300000E-01  0.300000E-01 -0.402724E-02  0.767260E-03  0.414450E-03  0.000000E+00  0.330000E-01  0.330000E-01 -0.213427E-02  0.269706E-03 -0.874962E-04  0.000000E+00 
LT:      20  0.600000E+02 -0.298473E+05 -0.174405E+00  0.191232E+00 -0.301680E-01 -0.201689E+00  0.198950E+00  0.780681E-02  0.300000E-01  0.300000E-01 -0.536475E-02  0.406582E-02 -0.143339E-03  0.000000E+00  0.315000E-01  0.315000E-01 -0.342832E-02  0.198384E-02 -0.832917E-03  0.000000E+00 
LT:      30  0.600000E+02 -0.298473E+05 -0.131307E+00  0.184406E+00 -0.241282E-01 -0.190284E+00  0.198411E+00 -0.182463E-02  0.300000E-01  0.300000E-01 -0.682818E-02  0.251581E-02  0.548651E-03  0.000000E+00  0.310000E-01  0.310000E-01 -0.441340E-02  0.221534E-02 -0.104723E-02  0.000000E+00 
LT:      40  0.600000E+02 -0.298473E+05 -0.122777E+00  0.171159E+00 -0.153756E-01 -0.170766E+00  0.198525E+00 -0.540674E-02  0.300000E-01  0.300000E-01 -0.113131E-01  0.171733E-02  0.143546E-02  0.000000E+00  0.307500E-01  0.307500E-01 -0.550866E-02  0.228385E-02 -0.681462E-03  0.000000E+00 
LT:      50  0.600000E+02 -0.298473E+05 -0.149361E+00  0.204416E+00  0.591725E-02 -0.166086E+00  0.194288E+00 -0.416044E-02  0.300000E-01  0.300000E-01 -0.165411E-01  0.536241E-02  0.926851E-04  0.000000E+00  0.306000E-01  0.306000E-01 -0.721382E-02  0.272763E-02 -0.544643E-03  0.000000E+00 
LT:      60  0.600000E+02 -0.298474E+05 -0.145456E+00  0.205595E+00 -0.161225E-02 -0.165575E+00  0.197047E+00 -0.168228E-02  0.300000E-01  0.300000E-01 -0.171287E-01  0.487046E-02  0.102017E-02  0.000000E+00  0.305000E-01  0.305000E-01 -0.892255E-02  0.309365E-02 -0.889064E-04  0.000000E+00 
LT:      70  0.600000E+02 -0.298474E+05 -0.214714E+00  0.217621E+00 -0.503011E-02 -0.168305E+00  0.199278E+00 -0.102739E-02  0.300000E-01  0.300000E-01 -0.204709E-01  0.393617E-02  0.367277E-02  0.000000E+00  0.304286E-01  0.304286E-01 -0.104365E-01  0.313277E-02  0.373321E-03  0.000000E+00 
LT:      80  0.600000E+02 -0.298473E+05 -0.166664E+00  0.231742E+00  0.181246E-02 -0.168733E+00  0.199834E+00  0.442097E-03  0.300000E-01  0.300000E-01 -0.224879E-01  0.690319E-02  0.472292E-02  0.000000E+00  0.303750E-01  0.303750E-01 -0.116710E-01  0.320249E-02  0.984400E-03  0.000000E+00 
LT:      90  0.600000E+02 -0.298474E+05 -0.191582E+00  0.193767E+00 -0.182747E-02 -0.170830E+00  0.201597E+00 -0.264020E-03  0.300000E-01  0.300000E-01 -0.245167E-01  0.516866E-02  0.298530E-02  0.000000E+00  0.303333E-01  0.303333E-01 -0.128471E-01  0.340855E-02  0.132651E-02  0.000000E+00 
LT:     100  0.600000E+02 -0.298474E+05 -0.209704E+00  0.199980E+00  0.396679E-02 -0.174729E+00  0.202748E+00 -0.170064E-02  0.300000E-01  0.300000E-01 -0.217354E-01  0.179831E-02 -0.545566E-03  0.000000E+00  0.303000E-01  0.303000E-01 -0.138473E-01  0.343450E-02  0.130823E-02  0.000000E+00 

Final values (basic output MC simulation)
Energy (eV):            -0.298474E+05
Polarization (C/m2):    -0.209704    0.199980    0.003967
Strain:                  0.030000    0.030000   -0.021735    0.001798   -0.000546    0.000000

-----------------------------------------------------------------
 Backing up the input read by fdf:

System_name     srtio3T60
Print_routineinfo         5     # default value
Print_routinedepth         0     # default value
Print_inout         0     # default value
Print_interactions         0     # default value
Print_k-vectors         0     # default value
Print_energies         1     # default value
Print_wavefunctions        -1     # default value
Print_electrostatic         1     # default value
Print_orbocc         1     # default value
Print_bands        -1     # default value
Print_forces        -1     # default value
Print_detail        -1     # default value
Print_Just_Geometry        -1     # default value
Print_Velocity         0     # default value
Print_Space_Hamiltonian        -1     # default value
Print_space_denmat        -1     # default value
Print_Geometry         1     # default value
Print_geoformat     normal     # default value
Print_test_file        -1     # default value
Print_local_lattice_energy        -1     # default value
Print_local_current        -1     # default value
Print_std_electron_nsteps         1     # default value
Print_std_lattice_nsteps             10
Print_std_energy              T
Print_std_polarization              T
Print_std_current              F     # default value
Print_std_strain              T
Print_std_av_polarization              T
Print_std_av_strain              T
Print_std_av_energy              F     # default value
Print_std_delta_energy              F     # default value
Print_std_temperature              T     # default value
Print_std_maximum_force              F     # default value
Print_std_maximum_stress              F     # default value
Print_std_mdd              T     # default value
Print_std_fermi              T     # default value
Print_std_pulay              T     # default value
Print_std_neh              T     # default value
Print_std_el_pol              T     # default value
Print_std_micro              F     # default value
run_mode     monte_carlo
Print_std_mdd              F     # default value
Print_std_fermi              F     # default value
Print_std_pulay              F     # default value
Print_std_neh              F     # default value
Print_std_electron_nsteps     10000     # default value
No_electron
%block fix_strain_component
 T T F F F T
T                                                                                             T
T                                                                                             T
F                                                                                             F
F                                                                                             F
F                                                                                             F
T                                                                                             T
Parameter_file     srtio3_full_lat.xml
%block Supercell
 2 2 2
%endblock Supercell
electric_field_scale          1.000000000     v/m     # default value
%block static_electric_field
 0.0 0.0 0.0
temperature         0.1000000000E-98 kelvin     # default value
%block external_stress
 0.0 0.0 0.0 0.0 0.0 0.0
Charge_tolerance         0.1000000000E-07     # default value
Geometry_restart
Geometry_restart     srtio3T60.restart
MC_nsweeps
MC_nsweeps            100
MC_temperature
MC_temperature          60.00000000         kelvin
# above item originally: MC_temperature          60.00000000     kelvin
MC_annealing_rate          1.000000000
MC_init_step_d         0.9448629925E-01 bohr     # default value
MC_init_step_s         0.1889725985E-02 bohr     # default value
MC_max_step_d         0.9448634389           bohr
# above item originally: MC_max_step_d         0.5000000000     ang
MC_min_step_d         0.1889725985E-05 bohr     # default value
MC_target_acc_d         0.5000000000         # default value
MC_target_acc_s         0.5000000000         # default value
MC_strains              T
n_write_mc
n_write_mc             10
Print_std_lattice_nsteps
*****************************************************************
 Memory usage:
  Function             Total_memory(Mb)  Max_memory(Mb)      Node
  setup_main                     0.0256          0.0256         0
  setup_basicvariables           0.0056          0.0056         0
  setup_latt_hamil               0.0014          0.0014         0
  setup_init_data_node           0.0256          0.0256         0
  hel_setup_structfactors        0.0256          0.0256         0
  hlt_create_supercell           0.0014          0.0014         0

 Timing statistics (seconds)
  Function                       Wall   Max_Wall    Max_CPU  Node
  setup_main                     3.73       3.73       3.73     0
  setup_readinput                3.51       3.51       3.51     0
  montecarlo_main                0.20       0.20       0.20     0
  montecarlo_sweep               0.20       0.20       0.20     0
  io_readparameterfile           3.51       3.51       3.51     0
  setup_latt_hamil               0.22       0.22       0.22     0
  hlt_add_harmonic_didi          0.22       0.22       0.22     0

 Total memory usage:     0.03 MB
 Total WALL:           3.936 seconds
*****************************************************************
*                                                               *
*               Succesfully ending SCALE-UP                     *
*                                                               *
*****************************************************************
